gftest.sh

```diff
@@ -1,41 +1,17 @@
 #
-# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# Copyright (c) 2020 Eclipse Foundation and/or its affiliates. All rights reserved.
```

Comment:
Strictly the copyright is yours if it is original work see https://www.eclipse.org/projects/handbook/#ip-copyright-headers you license the work to the Eclipse foundation under the EPL

Reply:
Okay, I'll keep that in mind for a next commit. I copied the loop from another test file, though I was the one (and only one) to have written that loop.